.github/workflows/sql-test-and-build-workflow.yml

```diff
@@ -32,13 +49,30 @@ jobs:
 
     # This step uses the codecov-action Github action: https://github.com/codecov/codecov-action
     - name: Upload SQL Coverage Report
-      uses: codecov/codecov-action@v1
+      if: always()
+      uses: codecov/codecov-action@v3
       with:
         flags: sql-engine
         token: ${{ secrets.CODECOV_TOKEN }}
 
     - name: Upload Artifacts
-      uses: actions/upload-artifact@v1
+      uses: actions/upload-artifact@v2
       with:
         name: opensearch-sql
         path: opensearch-sql-builds
+
+    - name: Upload test reports
```

Comment:
why add this? upload to where?

Reply:
It create a downloadable artifact, like you see there on bottom. Default lifetime of those is few days or one week.
https://github.com/Bit-Quill/opensearch-project-sql/actions/runs/3087022572